src/test/java/org/aksw/iguana/cc/tasks/storage/impl/CSVStorageTest.java

```diff
@@ -0,0 +1,201 @@
+package org.aksw.iguana.cc.tasks.storage.impl;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvException;
+import org.aksw.iguana.cc.config.IguanaConfig;
+import org.aksw.iguana.cc.tasks.stresstest.metrics.*;
+import org.aksw.iguana.cc.tasks.stresstest.metrics.impl.AggregatedExecutionStatistics;
+import org.aksw.iguana.cc.tasks.stresstest.metrics.impl.NoQ;
+import org.aksw.iguana.cc.tasks.stresstest.metrics.impl.NoQPH;
+import org.aksw.iguana.cc.tasks.stresstest.metrics.impl.QPS;
+import org.aksw.iguana.cc.tasks.stresstest.storage.impl.CSVStorage;
+import org.aksw.iguana.commons.rdf.IONT;
+import org.aksw.iguana.commons.rdf.IPROP;
+import org.aksw.iguana.commons.rdf.IRES;
+import org.apache.commons.io.FileUtils;
+import org.apache.jena.rdf.model.Model;
+import org.apache.jena.rdf.model.ModelFactory;
+import org.apache.jena.rdf.model.Resource;
+import org.apache.jena.rdf.model.ResourceFactory;
+import org.apache.jena.vocabulary.RDF;
+import org.apache.jena.vocabulary.RDFS;
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.io.FileReader;
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.math.BigInteger;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class CSVStorageTest {
+    CSVStorage storage;
+    Path folder;
+    Path suiteFolder;
+
+    @BeforeEach
+    public void setup() throws IOException {
+        this.folder = Files.createTempDirectory("iguana-CSVStorage-test");
+        this.suiteFolder = folder.resolve(IguanaConfig.getSuiteID());
+    }
+
+    public static Arguments createTestData1() {
```

Comment:
The data generation of this test is highly fragile in a sense that small changes to the rdf serialization render the generated data incorrec.t. It could be more robust if it relied on StresstestResultProcessor directly to generate a jena model. 
This applies in a similar fashion to the other tests in this package as well. This doesn't need to be patched but can be moved to an issue.